apps/web-client/src/lib/paths.ts

```diff
@@ -19,6 +19,7 @@ const PATHS = {
 
 	// Order mgmt
 	supplier_orders: `${basepath}/orders/suppliers/`,
+	supplier_order_status: `${basepath}/orders/suppliers/order/`,
```

Comment:
I'm not convinced by these routes, but I don't think we should worry too much about it here. Just leaving a note that I will write up a ticket for discussion